src/main/java/com/bwell/auth/model/Token.java

```diff
@@ -0,0 +1,31 @@
+package com.bwell.auth.model;
+
+public class Token {
+    private String accessToken;
+    private String expiresIn;
```

Comment:
should we change the `String` type to Java's DateTime class? 
cos the expiration info should be coming from the json response from the auth token server once a token is generated,

Reply:
We could, but the code will be more complex. The time on the token is millis since epoch, I think. So, it's possible. But, it's not really needed, as we just hard coded the expiration time of 60 mins. Mostly since it's short lived code.

Reply:
Let me know if you want it changed

Reply:
then sounds good "as is"